src/service/api/routes/search.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const {Router} = require(`express`);
+const router = new Router();
+const SearchController = require(`../controllers/search-controller`);
+
+module.exports = (app) => {
+  app.use(`/search`, router);
```

Comment:
```suggestion
const SearchController = require(`../controllers/search-controller`);

module.exports = (app) => {
const router = new Router();

  app.use(`/search`, router);
```
сделай так для всех роутов. тк дальше будет сложнее, и может быть неожиданное поведение местами